OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -86,12 +83,13 @@ private set
 			}
 		}
 
-		public float MinZoom => minZoom;
+		public float MinZoom { get; private set; } = 1f;
+		public float MaxZoom { get; private set; } = 2f;
```

Comment:
This exposes something that wasn't exposed before but I think that's fine.

Reply:
Yes, it seemed sensible to have both the min and max zoom available so I decided to make that change manually.